src/src/components/form/CustomSection.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <th
+    colspan="4"
+    class="sectionHeader"
+  >
+    <h3
+      @click="toggle($event)"
+      v-html="field.title"
+      :class="{'toggle' : field.canToggleSection}"
+    />
+    <span>
+      <a
+        v-if="field.canDeleteSection"
+        href="" 
+        @click="removeFields($event)"
+        class="list-btn icon-times"
+      />
+      <a
+        v-if="field.canToggleSection"
+        href=""
+        @click="toggle($event)"
+        class="list-icon"
+        :class="getToggleClass"
```

Comment:
Codacy has a fix for the issue: [Attribute ":class" should go before "@click".](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
          class="list-icon"
```